drivers/soundwire/intel.c

```diff
@@ -1386,7 +1373,7 @@ int intel_link_startup(struct auxiliary_device *auxdev)
 	}
 
 	/* Initialize shim, controller */
-	ret = intel_init(sdw);
+	ret = intel_link_power_up(sdw);
```

Comment:
@plbossart We had `intel_link_power_up` and `intel_shim_init` before, but now only `intel_link_power_up` is left.
We call `intel_link_power_down` and `intel_shim_wake` in power_down operations, but only `intel_link_power_up` in power_up operations.
Could you add explanation in the commit message?

Reply:
I am not following the comment @bardliao. What is the concern? is this that we have a single operation on power_up and two on power-down?

Reply:
Sorry @plbossart please forgive my poor eyes and ignore my comment. For some reason, I thought `intel_shim_init(sdw);` is added in `intel_shim_wake`.

Reply:
no worries @bardliao